docs/reference/eql/functions.asciidoc

```diff
@@ -8,8 +8,71 @@ experimental::[]
 
 {es} supports the following EQL functions:
 
+* <<eql-fn-endswith>>
 * <<eql-fn-substring>>
 
+[discrete]
+[[eql-fn-endswith]]
+=== `endsWith`
+
+Returns `true` if a source string ends with a provided substring.
```

Comment:
This function is case insensitive, as well. And I believe it's worth being mentioned in docs.

Reply:
Thanks! Addressed with 508be061d6be82a802863688765c46323f12283a.